label_studio/core/settings/base.py

```diff
@@ -255,7 +255,9 @@
     'SECURITY_DEFINITIONS': {
         'token': {
             'type': 'token',
-            'url': '/business/account/#/token'
+            'name': 'Token',
+            'in': 'header',
+            'url': '/user/account'
```

Comment:
Updated the URL to match where in the product the token gets retrieved and added more details here in the hopes that it would make the https://labelstud.io/api/#section/Authentication page more useful. I'm not able to build LS locally due to the Silicon chip so please make sure I didn't break anything by changing this!

Reply:
cc: @makseq please check this 😬

Reply:
@niklub you said that this change doesn't work, correct?